packages/core/src/recon.ts

```diff
@@ -169,11 +169,10 @@ export class ReconApi extends Observable<ReconEventFeedResponse> implements IRec
     }
     try {
       const headers = { 'Content-Type': 'application/json' }
-      const body = { ...(controller && { controller }) }
-      await this.#sendRequest(this.#url + `/ceramic/interests/model/${model.toString()}`, {
+      const query = controller ? `controller=${controller}` : ''
```

Comment:
Do we have a canonical way of constructing query strings for URLs? This string manipulation is not ideal.